.clj-kondo/config.edn

```diff
@@ -78,6 +84,7 @@
                                       schema.core/both
                                       {:namespaces ["metabase.*"]}}}
            :unresolved-var {:exclude [colorize.core]}
+           :unsorted-required-namespaces {:level :warning}
```

Comment:
@borkdude 's change from the closed `https://github.com/metabase/metabase/pull/21512`